santabot/cogs/give.py

```diff
@@ -144,7 +148,8 @@ async def __do_gifting(
         await ctx.trigger_typing()
 
         # Send a present from one User to another
-        if isinstance(recipient, discord.Member):
+        if isinstance(recipient, discord.Member) \
+                and recipient.id != invoking_user.id:
```

Comment:
I think it would be better to move this check down a level, like:

```python
if isinstance(recipient, discord.Member):
    if recipient.id == invoking_user.id:
        await ctx.send('Nice try {invoking_user.mention}...')
        return

    # cooldown...

Reply:
I didn't even think of it as a trick so much as a mistake

Reply:
Yeah it was but it's also a funny condition to account for

Reply:
should it cancel the gift entirely?